packages/hub/src/lib/file-download-info.ts

```diff
@@ -48,12 +48,14 @@ export async function fileDownloadInfo(params: {
 
 	const resp = await (params.fetch ?? fetch)(url, {
 		method: "GET",
-		headers: params.credentials
-			? {
-					Authorization: `Bearer ${params.credentials.accessToken}`,
-					Range: "bytes=0-0",
-			  }
-			: {},
+		headers: {
+			...(params.credentials
+				? {
+						Authorization: `Bearer ${params.credentials.accessToken}`,
+				  }
+				: undefined),
```

Comment:
```suggestion
			...(params.credentials && {
						Authorization: `Bearer ${params.credentials.accessToken}`,
				  }),
```

In this codebase you can do this if you want :p

Reply:
I like that :D